plan/new_plan_test.go

```diff
@@ -903,6 +903,35 @@ func (s *testPlanSuite) TestConstantFolding(c *C) {
 	UseNewPlanner = false
 }
 
+func (s *testPlanSuite) TestCoveringIndex(c *C) {
+	cases := []struct {
+		columnNames []string
+		indexNames  []string
+		indexLens   []int
+		isCovering  bool
+	}{
+		{[]string{"a"}, []string{"a"}, []int{-1}, true},
+		{[]string{"a"}, []string{"a", "b"}, []int{-1, -1}, true},
+		{[]string{"a", "b"}, []string{"b", "a"}, []int{-1, -1}, true},
+		{[]string{"a", "b"}, []string{"b", "c"}, []int{-1, -1}, false},
```

Comment:
Add the test case as following: `{[]string{"a", "b"}, []string{"a", "c"}, []int{-1, -1}, false},`